pocs/observatory.py

```diff
@@ -70,16 +77,25 @@ def __init__(self, *args, **kwargs):
 # Properties
 ##########################################################################
 
-    @property
-    def is_dark(self):
-        horizon = self.location.get('twilight_horizon', -18 * u.degree)
+    def is_dark(self, horizon='astro'):
+
+        try:
+            horizon_deg = self.config['location']['{}_horizon'.format(horizon)]
+        except KeyError:
+            # Look for legacy name. Note this will happen for either horizon
+            try:
+                horizon_deg = self.config['location']['twilight_horizon']
+            except KeyError:
+                self.logger.info("Can't find twilight_horizon, using -18°")
```

Comment:
This isn't in sync with the yaml file, which has no astro_horizon. Did you mean to add one?

Reply:
Yes, meant to switch, thanks.